prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'Peter Jarl Victor';
+    console.log('value of fullName:', fullName);
+    var numberOfPets = 'One';
+    console.log('value of numberOfPets:', numberOfPets);
+    var likesVegetables = true;
+    console.log('value of likesVegetables:', likesVegetables);
+    var nothing = 'null';
```

Comment:
Here we don't want a string with the word `'null'` in it we want to use just `null` with no quotes.